rpc/server/handlers.go

```diff
@@ -332,7 +337,8 @@ func (wsc *WSConnection) readRoutine() {
 				} else {
 					log.Notice("Subscribe to event", "id", wsc.id, "event", event)
 					wsc.evsw.AddListenerForEvent(wsc.id, event, func(msg interface{}) {
-						wsc.writeRPCResponse(NewRPCResponse(request.Id, RPCEventResult{event, msg}, ""))
+						// NOTE: RPCResponses of subscribed events have id suffix "#event"
+						wsc.writeRPCResponse(NewRPCResponse(request.Id+"#event", RPCEventResult{event, msg}, ""))
```

Comment:
Yeah.

I guess the problem is that it still doesn't solve the unknown-type problem if say you subscribe to a wildcard like Acc/XYZ/* (not that we support these yet).  So maybe we should just add the type-byte.